### DISCUSSION TITLE
How to make UI changes per platform and screen size?

### DISCUSSION THREAD

**Discussion by TonyK-FmGroup** [dotnet/maui#7812] [Q&A]
What is the recommended way to create a differences in the UI per platform and screen size? For example, an application on a windows device with a 27 inch screen can be very different to a mobile phone or a tablet. 
Can they share a main XAML page with differences in layout? Do we need to create a whole different start page?

**Comment by xtuzy** [comment]
For different platform or screen size, you can see
1. https://docs.microsoft.com/en-us/dotnet/maui/xaml/pass-arguments#specify-a-generic-type-argument
2. SizeChanged event
3. DataTemplateSelector
....
